azure_pipelines/spm-framework.yml

```diff
@@ -42,7 +42,7 @@ jobs:
     clean: true
```

Comment:
This pull request does not update CHANGELOG.md.

Please consider if this change would be noticeable to a partner or user and either update CHANGELOG.md or resolve this conversation.
<!-- changelog-comment -->